charters/DataStore/charter.md

```diff
@@ -0,0 +1,72 @@
+
+## Objective
+The objective of the Data Store group is to deliver a versioned immutable object based data repository that is highly available and scalable. Data will be replicated to at least two commercial clouds (Amazon and Google). Data will be accessible through a variety of programatic interfaces as well as a command line interface.
+
+## In-scope
+
+### Interfaces
+* Data Store read and write APIs for data (PUT bundle, PUT file, GET bundle, GET file) - maintenance and extension of the implementation of the basic data access APIs.
+* Maintain and extend the **Checkout service API** which enables data copy to a local filesystem or a personal cloud environment
+* Maintain and extend the **Collections service API** to do basic operations on arbitrary collections of objects in the Data Store.
+* Publish API documentation and examples for both the Data Store REST interface and Python bindings.
+* The **Command Line Interface** (CLI) is a foundational tool for interacting with the DCP. The Data Store team is responsible for the specific Data Store commands and the maintenance of the infrastructure that allows other services such as Upload and Ingest to integrate their commands into the CLI.
+
+### Core capabilities
+* Maintain and extend the DSS data model and lifecycle (such as versioned bundles)
+* Transition Data Store Subscriptions/Eventing services from the current dependence on Elastic Search Percolate to the AWS and GCP cloud infrastructure.
+* Multi-cloud replication of objects
+   1. Maintenance and improvements to the synchronization implementation between AWS and GCP
+   2. Document interfaces to enable new cloud implementations by 3rd parties 
+   3. Supporting multiple replicas within a single cloud
+* Support for pluggable indexes - Define a standard interface to enable pluggable indexing modules to receive Data Store events
+
+### Security
+* User authentication system implementation
+* Data access authorization system implementation 
+* DevSecOps - implementation of features required for eventual FISMA moderate deployments (authentication, authorization, logging, auditing, etc).
+* Operations for Data Store - Implement and configure tools to facilitate the operation of the Data Store service in a production environment
+
+### Community engagement
+* Triage and integration of feature requests from the community into the Data Store roadmap. 
+* Review and acceptance process for third party software contributions through pull requests
+* Outreach and engagement of the community
+* Offer trainings on how to contribute to the Data Store project, or reuse it for a different project.
+* Host hackathons for extending the Data Store feature set. 
+
+## Out-of-scope
```

Comment:
Since we're also simultaneously writing charters for Matrix and Query, this section feels a bit redundant? It was more critical during the refactoring conversations to sketch and clarify division of responsibilities.

Reply:
Leaving in the index/query for now, just to be clear. I think it helps because the ES code has been part of the Data Store set of responsibilities until now.